src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -142,9 +143,6 @@ func Run(updateHandler func(updated *Config)) error {
 
 func updateGlobals(cfg *Config) error {
 	globals.InstanceId = cfg.InstanceId
-	loc := &geolookup.City{}
-	loc.Country.IsoCode = cfg.Country
-	globals.SetLocation(loc)
```

Comment:
> save location across runs so the user dot doesn't start in null island until first geolocation request.

That one actually makes sense to me. Let me see:

1. If that's actually supported now
1. If I can just add it